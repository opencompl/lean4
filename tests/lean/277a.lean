set_option hygiene false
infixl:67 " <>< "  => nonexistent

#eval (1 <>< 11 : UInt64)
