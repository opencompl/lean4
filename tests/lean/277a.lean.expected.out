277a.lean:4:7-4:15: error: unknown identifier 'nonexistent'
